packages/core/strapi/package.json

```diff
@@ -130,7 +130,8 @@
     "uuid": "^3.3.2"
   },
   "devDependencies": {
-    "supertest": "^6.1.6"
+    "supertest": "^6.1.6",
+    "typescript": "4.6.2"
```

Comment:
Shouldn't we add this dependencies only in the typescript projects I am guessing that we don't need them in JSs ones.

Reply:
This dependency isn't linked to a strapi project but to the strapi core. We need typescript to write our toolchain (custom compilers)